output/components.eslint.txt

```diff
@@ -2,6 +2,9 @@
 
 The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.
 
+/home/travis/build/Talend/ui/packages/components/src/AboutDialog/index.js
+  1:8  error  Using exported name 'AboutDialog' as identifier for default export  import/no-named-as-default
```

Comment:
Fix lint error please. You can, instead of exporting the component for tests, use the default import and instanciate `<AboutModal.WrappedComponent>`

Reply:
Fixed, thanks